packages/peregrine/lib/store/reducers/checkout.js

```diff
@@ -15,7 +15,7 @@ const initialState = {
     receipt: {
         order: {}
     },
-    shippingAddress: null,
+    shippingAddress: {},
```

Comment:
I may be able to revert this change, though we probably can default this to `{}` safely as it is set to that in `beginCheckout`.